app/code/Magento/Eav/Test/Unit/Model/Api/SearchCriteria/CollectionProcessor/FilterProcessorTest.php

```diff
@@ -144,25 +151,24 @@ public function testProcess()
         $model->process($searchCriteriaMock, $collectionMock);
     }
 
-    /**
-     * @expectedException \InvalidArgumentException
-     */
     public function testProcessWithException()
     {
-        /** @var \stdClass|\PHPUnit_Framework_MockObject_MockObject $customFilterMock */
-        $customFilterMock = $this->createPartialMock(\stdClass::class, ['apply']);
+        $this->expectException(InvalidArgumentException::class);
```

Comment:
Please add the assertion of an exception right before the original methods call.
E.g.
```
$model->process($searchCriteriaMock, $collectionMock);
$this->expectException(InvalidArgumentException::class);
```
Otherwise, some unexpected exception can be interpreted incorrectly.